scripts/kind-e2e/e2e.sh

```diff
@@ -191,14 +227,12 @@ function test_with_e2e_tests {
 function delete_subm_pods() {
     context=$1
     ns=$2
-    if kubectl --context=$context wait --for=condition=Ready pods -l app=submariner-engine -n $ns --timeout=60s > /dev/null 2>&1; then
-        echo Removing submariner engine pods...
-        kubectl --context=$context delete pods -n submariner -l app=submariner-engine
-    fi
-    if kubectl --context=$context wait --for=condition=Ready pods -l app=submariner-routeagent -n $ns --timeout=60s > /dev/null 2>&1; then
-        echo Removing submariner route agent pods...
-        kubectl --context=$context delete pods -n submariner -l app=submariner-routeagent
-    fi
+    for app in submariner-engine submariner-routeagent submariner-globalnet; do
```

Comment:
I would rather get the app name in the parameters and then do the iteration from outside the function, this makes it more reusable.

Reply:
Done